src/Common/tendency.jl

```diff
@@ -62,7 +89,7 @@ end
     @. ts = TD.PhaseNonEquil_ρTq(thermo_params, ρ, T, PP(q_tot, q_liq, q_ice))
     @. p = TD.air_pressure(thermo_params, ts)
     @. θ_dry = TD.dry_pottemp(thermo_params, T, ρ_dry)
-    @. θ_liq_ice = TD.liquid_ice_pottemp(thermo_params, ts)
+    @. θ_liq_ice = θ_dry #TD.liquid_ice_pottemp(thermo_params, ts) - prevents error...
```

Comment:
Why did you change this line?

Reply:
I wonder if the fact that we are running it with q_vap = 0 is the problem

Reply:
Just changed it back to using θ_liq_ice from Thermodynamics, and no error! So whatever was causing it when I originally commented it out must be fixed now